Geometry/HGCalCommonData/src/HGCalWaferMask.cc

```diff
@@ -6,7 +6,7 @@
 #include <algorithm>
 #include <sstream>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
@bsunanda , you need to comment this out.

Reply:
Done

>